content/articles/integrated-domain-providers.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Integrated Domain Providers at DNSimple
+excerpt: Link an integrated domain provider to your DNSimple account to manage domains at other domain providers, from within DNSimple.
+categories:
+- Domains
+- Integrations
+---
+
+# Integrated Domain Providers
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple supports the ability to manage domains that are registered with other domain providers external to DNSimple. DNSimple calls such providers "integrated domain providers".
+
+## Supported Features
+
+- **View domain information**: When you link an integrated domain provider to your DNSimple account, domains registered via that integrated domain provider will be imported into DNSimple and listed on the Domains page.
+- **Domain expiration notices**: Get email notifications when your integrated domains are near expiry.
+
+## Supported Integrated Domain Providers
+
+To manage integrated domains from DNSimple, you first have to link an integrated domain provider to your DNSimple account.
+
+DNSimple supports domain integration with the providers listed below.
+
+- [GoDaddy](https://www.godaddy.com)
+
+## Linking an Integrated Domain Provider to your account
+
+1. At DNSimple, go to the <label>Account</label> page, and click the <label>Integrated Providers</label> tab.
+1. Under <label>Add an integrated provider<label>, click the link button for the integrated domain provider you want to link to your DNSimple account.
+![Link an integrated domain provider](/files/account-integrated-domain-provider-link.png)
```

Comment:
Can you use the image from production (with the GoDaddy logo that will be seen in the app)?

Reply:
This screenshot was taken from Sandbox, but I believe it is the same logo asset, what do you think?

Reply:
I may have commented in the wrong place. This is the one I was referring to:

https://github.com/dnsimple/dnsimple-support/pull/1065/files#diff-76c0da71781a76b5a2dcaab7777b2e6f8c301710f07209156dd5e42476d04f91

Reply:
Ah I see! Fixed here: https://github.com/dnsimple/dnsimple-support/pull/1065/commits/866426689b7ed204b9dd86a8591d5d45ba4446e4 🎉